vendor/knative.dev/pkg/injection/sharedmain/main.go

```diff
@@ -156,6 +166,10 @@ func MainNamed(ctx context.Context, component string, ctors ...injection.NamedCo
 	// HACK: This parses flags, so the above should be set once this runs.
 	cfg := injection.ParseAndGetRESTConfigOrDie()
 
+	// Split the disabled controllers string and store them
+	disabledControllersList := strings.Split(*disabledControllers, ",")
+	currentDisabledControllers = disabledControllersList
```

Comment:
why not directly?
```suggestion
	currentDisabledControllers = strings.Split(*disabledControllers, ",")
```

Reply:
does not really matter - is anyways just a tmp fix

/unhold